### ISSUE TITLE
LearnedPGD

### ISSUE THREAD

**Issue by nducros** [openspyrit/spyrit#220]
See [here](https://spyrit.readthedocs.io/en/master/_autosummary/spyrit.core.recon.LearnedPGD.html#spyrit.core.recon.LearnedPGD)

* Examples are copied from Pinv (not up to date)
* @jabascal Do you confirm we can remove `reconstruct_expe`?

**Comment by nducros** [comment]
Yes. Most methods of the class have the same errors.

**Comment by jabascal** [comment]
@ducros

Not needed for new classes created. At fig_recon_sim_prelim.py, I am doing


                    if net_arch == 'lpgd' or net_arch == 'dfb-net':
                        rec_gpu = model.reconstruct(m, exp=True)
                    else:
                        model.prep.set_expe()
                        rec_gpu = model.reconstruct_expe(m)
                    rec = rec_gpu.cpu().detach().numpy().squeeze()
